main.ts

```diff
@@ -18,6 +18,8 @@ input.onButtonPressed(Button.B, () => {
 
 bluetooth.onBluetoothDisconnected(() => {
   connected = false;
+  gigglebot.motorPowerAssign(gigglebotWhichMotor.Left, 0);
+  gigglebot.motorPowerAssign(gigglebotWhichMotor.Right, 0);
```

Comment:
Nitpick: that enum has a `gigglebot.WhichMotor.Both` that could reduce this to one line.

Reply:
One command better than two :+1:. Done.